src/bench/rpc_blockchain.cpp

```diff
@@ -15,7 +15,7 @@
 namespace {
 
 struct TestBlockAndIndex {
-    TestingSetup test_setup{};
+    const std::unique_ptr<const TestingSetup> testing_setup{MakeNoLogFileContext<const TestingSetup>(CBaseChainParams::MAIN)};
```

Comment:
This seems kind of unruly.

Reply:
> unruly

C++ requires this. (You can't use `auto` for non-static members)